lookup_plugin/s3_handler.py

```diff
@@ -482,19 +683,26 @@ def start_gcService_process(cluster_params, dirName, dryRun, delDBO):
 
     # Open the log file to pass the fp to subprocess.Popen
     fp = open(gcLogFile, "a+")
-
-    path = get_dir_path(cluster_params, dirName)
     bin_path = '%s/GCService' % binary_dir
-    matches = re.findall(r'[\w-]{36}', path)
-    vdev_uuid = matches[-1] if matches else None
 
     cmd = []
     s3config = '%s/s3.config.example' % binary_dir
     # Prepare path for log file.
     s3LogFile = "%s/%s/s3Download" % (base_dir, raft_uuid)
-    downloadPath = "%s/%s/gc-downloaded-obj" % (base_dir, raft_uuid)
+    
+    if partition:
+        downloadPath = "%s/%s/gc/gc_download" % (base_dir, raft_uuid)
+    else:
+        downloadPath = "%s/%s/gc-downloaded-obj" % (base_dir, raft_uuid)
+        if not os.path.exists(downloadPath):
+            # Create the directory path
+            try:
+                os.mkdir(downloadPath, 0o777)
+            except Exception as e:
+                print(f"An error occurred while creating '{downloadPath}': {e}")
+    
     cmd = [bin_path, '-path', downloadPath, '-s3config', s3config, '-s3log', s3LogFile, '-t', '120',
-              '-l', '2', '-p', '7500', '-b', 'paroscale-test']
+              '-l', '4', '-p', '7500', '-b', 'paroscale-test']
```

Comment:
can you change it back to 2 , because it may take more space if we have more info prints